exercises/D-strings/exercise.js

```diff
@@ -1,3 +1,15 @@
 // Start by creating a variable `message`
 
-console.log(message);
+//Write a program that logs a message and its type
+const number = 4;
+let message = "Rahwa";
+let text;
+//Terminal shows a number
+console.log(typeof number);
+
+//Terminal shows a string
+console.log(typeof message);
+
+//Terminal shows undefined
+console.log(typeof text);
```

Comment:
You also need to output the variables too not just their type. In the example, they missed it too.